node-js/src/controllers/login/login-controller.js

```diff
@@ -4,8 +4,11 @@ const LoginController = {
     doLogin: async (req, res, next) => {
         try {
             const service = new LoginService();
-            const {status, data} = await service.getToken(req.body);
-            res.status(status).json({message:data});
+            const data = await service.getToken(req.body);
+            if (data.access_token === undefined || data.id_token === undefined || data.refresh_token === undefined || data.access_token === null || data.id_token === null || data.refresh_token === null) {
```

Comment:
Aqui `undefined` e `null` são [falsy](https://developer.mozilla.org/pt-BR/docs/Glossary/Falsy). Podemos remover a comparação.

```suggestion
            if (!data.access_token || !data.id_token || !data.refresh_token) {
```

O único ponto de atenção é se algum `falsy` for um valor válido para o token, como `0`, `false` ou string vazia.